graphql/resolvers.go

```diff
@@ -881,6 +888,32 @@ func (r *queryResolver) Patch(ctx context.Context, id string) (*restModel.APIPat
 	return patch, nil
 }
 
+func (r *queryResolver) Project(ctx context.Context, id string) (*restModel.APIProjectRef, error) {
+	project, err := r.sc.FindProjectById(id)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding project by id %s: %s", id, err.Error()))
+	}
+	apiProjectRef := restModel.APIProjectRef{}
+	err = apiProjectRef.BuildFromService(project)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error building APIProject from service: %s", err.Error()))
+	}
+	return &apiProjectRef, nil
+}
+
+func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProjectRef, patchesInput PatchesInput) (*Patches, error) {
+	patches, count, err := r.sc.FindPatchesByProjectPatchNameStatusesCommitQueue(*obj.Identifier, patchesInput.PatchName, patchesInput.Statuses, patchesInput.IncludeCommitQueue, patchesInput.Page, patchesInput.Limit)
```

Comment:
This should use Id, not Identifier (for context, we made a change with projects for IDs to be the global form of tracking projects across data models, so that Identifier can be modifiable in the future, so the ID should be used for a patches query. "FindProjectById" checks both ID and Identifier though)